LoadVariationRiskBalancingPlugin enhanced integration test

```
--- FAIL: TestLoadVariationRiskBalancingPlugin (2.17s)
    loadVariationRiskBalancing_test.go:158: Creating Pod "pod-1"
    loadVariationRiskBalancing_test.go:158: Creating Pod "pod-2"
    loadVariationRiskBalancing_test.go:175: 
        	Error Trace:	loadVariationRiskBalancing_test.go:175
        	Error:      	Not equal: 
        	            	expected: "node-3"
        	            	actual  : "node-1"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-node-3
        	            	+node-1
        	Test:       	TestLoadVariationRiskBalancingPlugin
E0712 11:24:55.730394   34462 collector.go:148] "Load watcher client failed" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:24:55.730418   34462 collector.go:94] "Unable to update metrics" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:25:25.729530   34462 collector.go:148] "Load watcher client failed" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:25:25.729559   34462 collector.go:94] "Unable to update metrics" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:25:55.730145   34462 collector.go:148] "Load watcher client failed" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:25:55.730222   34462 collector.go:94] "Unable to update metrics" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:26:25.729347   34462 collector.go:148] "Load watcher client failed" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:26:25.729398   34462 collector.go:94] "Unable to update metrics" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:26:55.730188   34462 collector.go:148] "Load watcher client failed" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:26:55.730239   34462 collector.go:94] "Unable to update metrics" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:27:02.230699   34462 framework.go:1000] "Failed running Bind plugin" err="pods \"p\" not found" plugin="DefaultBinder" pod="integration-test-be5b574e-9997-45db-bc2e-6afedd357877/p"
I0712 11:27:02.230784   34462 factory.go:231] "Pod doesn't exist in informer cache" pod="integration-test-be5b574e-9997-45db-bc2e-6afedd357877/p" err="pod \"p\" not found"
E0712 11:27:02.234210   34462 scheduler.go:322] "Error updating pod" err="pods \"p\" not found" pod="integration-test-be5b574e-9997-45db-bc2e-6afedd357877/p"
E0712 11:27:24.692627   34462 preemption_toleration.go:150] "Failed to parse preemption toleration policy of victim candidate's priorityclass.  This victim candidate can't tolerate the preemption" err="strconv.ParseInt: parsing \"a\": invalid syntax" PreemptorPod="integration-test-74074d07-8f65-4e4a-8bf4-6ebc472daa33/p" VictimCandidatePod="integration-test-74074d07-8f65-4e4a-8bf4-6ebc472daa33/victim-candidate" VictimCandidatePriorityClass="priority-class-for-victim-candidates-7"
E0712 11:27:25.729786   34462 collector.go:148] "Load watcher client failed" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:27:25.729904   34462 collector.go:94] "Unable to update metrics" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
FAIL
E0712 11:27:55.729931   34462 collector.go:148] "Load watcher client failed" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
E0712 11:27:55.729978   34462 collector.go:94] "Unable to update metrics" err="Get \"http://127.0.0.1:33367/watcher\": dial tcp 127.0.0.1:33367: connect: connection refused"
FAIL	sigs.k8s.io/scheduler-plugins/test/integration	877.652s
```

--- a/test/integration/loadVariationRiskBalancing_test.go
+++ b/test/integration/loadVariationRiskBalancing_test.go
@@ -61,8 +61,13 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 					Metrics: []watcher.Metric{
 						{
 							Type:     watcher.CPU,
-							Value:    10,
-							Operator: watcher.Latest,
+							Value:    30,
+							Operator: watcher.Average,
+						},
+						{
+							Type:     watcher.CPU,
+							Operator: watcher.Std,
+							Value:    20,
 						},
 					},
 				},
@@ -70,8 +75,13 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 					Metrics: []watcher.Metric{
 						{
 							Type:     watcher.CPU,
-							Value:    60,
-							Operator: watcher.Latest,
+							Value:    70,
+							Operator: watcher.Average,
+						},
+						{
+							Type:     watcher.CPU,
+							Operator: watcher.Std,
+							Value:    20,
 						},
 					},
 				},
@@ -79,8 +89,13 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 					Metrics: []watcher.Metric{
 						{
 							Type:     watcher.CPU,
-							Value:    0,
-							Operator: watcher.Latest,
+							Value:    50,
+							Operator: watcher.Average,
+						},
+						{
+							Type:     watcher.CPU,
+							Operator: watcher.Std,
+							Value:    50,
 						},
 					},
 				},
@@ -114,9 +129,6 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 		},
 	})
 
-	ns := fmt.Sprintf("integration-test-%v", string(uuid.NewUUID()))
-	createNamespace(t, testCtx, ns)
-
 	testCtx = initTestSchedulerWithOptions(
 		t,
 		testCtx,
@@ -127,7 +139,9 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 	go testCtx.Scheduler.Run(testCtx.Ctx)
 	defer cleanupTest(t, testCtx)
 
-	var nodes []*v1.Node
+	ns := fmt.Sprintf("integration-test-%v", string(uuid.NewUUID()))
+	createNamespace(t, testCtx, ns)
+
 	nodeNames := []string{"node-1", "node-2", "node-3"}
 	capacity := map[v1.ResourceName]string{
 		v1.ResourceCPU:    "2",
@@ -135,9 +149,8 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 	}
 	for i := 0; i < len(nodeNames); i++ {
 		node := st.MakeNode().Name(nodeNames[i]).Label("node", nodeNames[i]).Capacity(capacity).Obj()
-		node, err := cs.CoreV1().Nodes().Create(testCtx.Ctx, node, metav1.CreateOptions{})
+		_, err := cs.CoreV1().Nodes().Create(testCtx.Ctx, node, metav1.CreateOptions{})
 		assert.Nil(t, err)
-		nodes = append(nodes, node)
 	}
 
 	var newPods []*v1.Pod
@@ -163,7 +176,7 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 	}
 	defer cleanupPods(t, testCtx, newPods)
 
-	expected := [2]string{nodeNames[2], nodeNames[2]}
+	expected := [2]string{nodeNames[0], nodeNames[0]}
 	for i := range newPods {
 		err := wait.Poll(1*time.Second, 10*time.Second, func() (bool, error) {
 			return podScheduled(cs, newPods[i].Namespace, newPods[i].Name), nil

--- a/test/integration/loadVariationRiskBalancing_test.go
+++ b/test/integration/loadVariationRiskBalancing_test.go
@@ -89,13 +89,8 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 					Metrics: []watcher.Metric{
 						{
 							Type:     watcher.CPU,
-							Value:    50,
+							Value:    30,
 							Operator: watcher.Average,
-						},
-						{
-							Type:     watcher.CPU,
-							Operator: watcher.Std,
-							Value:    50,
 						},
 					},
 				},
@@ -176,7 +171,7 @@ func TestLoadVariationRiskBalancingPlugin(t *testing.T) {
 	}
 	defer cleanupPods(t, testCtx, newPods)
 
-	expected := [2]string{nodeNames[0], nodeNames[0]}
+	expected := [2]string{nodeNames[2], nodeNames[2]}
 	for i := range newPods {
 		err := wait.Poll(1*time.Second, 10*time.Second, func() (bool, error) {
 			return podScheduled(cs, newPods[i].Namespace, newPods[i].Name), nil

